docs/source/api/v4/server_capabilities.rst

```diff
@@ -50,6 +50,7 @@ Request Structure
 Response Structure
 ------------------
 :name:        The name of this :term:`Server Capability`
+:description: The description of this :term:`Server Capability`
```

Comment:
This should have a `..versionadded::` directive specifying that this was added in 4.1 - because it can't go into 4.0